tests/e2e/mnist_raycluster_sdk_test.py

```diff
@@ -14,30 +14,24 @@
 
 import pytest
 
-from support import random_choice, get_ray_image
+from support import *
 
 # Creates a Ray cluster, and trains the MNIST dataset using the CodeFlare SDK.
 # Asserts creation of AppWrapper, RayCluster, and successful completion of the training job.
 # Covers successfull installation of CodeFlare-SDK
 
 
+@pytest.mark.kind
+@pytest.mark.openshift
 class TestMNISTRayClusterSDK:
     def setup_method(self):
-        # Load the kube config from the environment or Kube config file.
-        config.load_kube_config()
-
-        # Initialize Kubernetes client
-        self.api_instance = client.CoreV1Api()
-        self.custom_api = kubernetes.client.CustomObjectsApi(
-            self.api_instance.api_client
-        )
+        initialize_kubernetes_client(self)
 
     def teardown_method(self):
-        if hasattr(self, "namespace"):
-            self.api_instance.delete_namespace(self.namespace)
+        delete_namespace(self)
 
     def test_mnist_ray_cluster_sdk(self):
-        self.create_test_namespace()
+        create_namespace(self)
         self.run_mnist_raycluster_sdk()
 
     def create_test_namespace(self):
```

Comment:
I believe this whole function can be removed as it is now in the `support.py` file.